driverkit/utils/checkfiles

```diff
@@ -8,6 +8,20 @@ source "${currentdir}/parseyaml"
 input="$1"
 filename="${input##*/}"
 
+# TODO: rm once old format support is dropped; just use arch="${input_parts[2]}"
+function get_arch_from_path() {
+    # Check if arch subdir exists
+    IFS=/ read -a input_parts <<< ${1}
+    if [[ ${#input_parts[@]} -eq 4 ]]; then
+        arch="${input_parts[2]}/"
+    else
+        arch=""
+    fi
+    echo -n "${arch}"
+}
+
+arch="$(get_arch_from_path ${input})"
```

Comment:
Fetch arch from config file path (if any); it is later used to check that if config file is under a $arch subfolder, its output module and probe must be under same $arch subfolder.